tests/runtime/utils/test_servicer_util.py

```diff
@@ -110,7 +111,9 @@ def test_service_util_validate_caikit_library_class_method_exists_does_raise():
 
 
 # ---------------- Tests for build_proto_response  --------------------------
-def test_servicer_util_build_proto_response_raises_on_garbage_response_type():
+def test_servicer_util_build_proto_response_raises_on_garbage_response_type(
+    sample_inference_service,
```

Comment:
Is this so that it's not relying on side effects?

Reply:
A good question...I think I accidentally added this in here when meaning to just add it to the below method